app/forms/decidim/registration_form.rb

```diff
@@ -73,7 +73,7 @@ def list_address_ids
     end
 
     def query_api
-      url = URI("https://api-adresse.data.gouv.fr/search/?q=#{address}")
+      url = URI("https://api-adresse.data.gouv.fr/search/?q="+CGI.escape(address))
```

Comment:
Bonjour Sébastien et merci pour la solution à ce problème ! 
Pour passer les tests rubocop, il faudrait seulement ajouter des espaces autour du "+". 

```suggestion
      url = URI("https://api-adresse.data.gouv.fr/search/?q=" + CGI.escape(address))
```